setup.py

```diff
@@ -108,6 +108,10 @@ def get_git_sha():
         'wtforms-json',
     ],
     extras_require={
+        'bigquery': [
+            'pybigquery',
```

Comment:
We might want to say `>={current_version}, <{semantic_version_that_may_break_backward_compatibility}` for both libs there.

Reply:
Done.